image/cli/mascli/templates/pipelinerun.yaml

```diff
@@ -175,6 +190,12 @@ spec:
       value: '$MAS_CATALOG_SOURCE'
     - name: mas_catalog_version
       value: '$MAS_CATALOG_VERSION'
+    - name: mas_add_channel
+      value: '$MAS_ADD_CHANNEL'
+    - name: mas_add_catalog
+      value: '$MAS_ADD_CATALOG'
+    - name: mas_catalog_version
```

Comment:
duplicate of line 191-192 above

Reply:
oh thanks! i'll remove that and also:
```
    - name: mas_catalog_version
      value: '$MAS_CATALOG_VERSION'
```

which is also duplicate of line 191